numba/parfors/parfor_lowering.py

```diff
@@ -1777,8 +1777,8 @@ def load_potential_tuple_var(x):
     # ----------------------------------------------------------------------------
     # Next, we prepare the individual dimension info recorded in gu_signature
     sig_dim_dict = {}
-    occurances = []
-    occurances = [sched_sig[0]]
+    occurrences = []
+    occurrences = [sched_sig[0]]
```

Comment:
Not to be fixed in this PR, but I'm not sure this makes sense, it's direct assignment to the same variable name made in adjacent statements.